pkg/api/v1/types.go

```diff
@@ -2216,6 +2209,12 @@ type PodStatus struct {
 	// More info: https://github.com/kubernetes/kubernetes/blob/master/docs/design/resource-qos.md
 	// +optional
 	QOSClass PodQOSClass `json:"qosClass,omitempty" protobuf:"bytes,9,rep,name=qosClass"`
+	// The list has one entry per init container in the manifest. The most recent successful
+	// init container will have ready = true, the most recently started container will have
+	// startTime set.
+	// Init containers are in alpha state and may change without notice.
```

Comment:
after this pr, this would no longer be the case.  remove this line about alpha state.